client/modules/IDE/components/ErrorModal.unit.test.jsx

```diff
@@ -8,20 +8,30 @@ jest.mock('../../../i18n');
 
 describe('<ErrorModal />', () => {
   it('renders type forceAuthentication', () => {
-    render(<ErrorModal type="forceAuthentication" closeModal={jest.fn()} />);
+    render(
+      <ErrorModal
+        type="forceAuthentication"
+        closeModal={jest.fn()}
+        service="google"
+      />
+    );
 
     expect(screen.getByText('Login')).toBeVisible();
     expect(screen.getByText('Sign Up')).toBeVisible();
   });
 
   it('renders type staleSession', () => {
-    render(<ErrorModal type="staleSession" closeModal={jest.fn()} />);
+    render(
+      <ErrorModal type="staleSession" closeModal={jest.fn()} service="google" />
+    );
```

Comment:
Personally I would say that the more logical fix for the PropTypes error would be to make it so that the `service` is not a required prop.  It's only needed when `type="oauthError"` and it's not used in any other circumstance so it's kind of weird that it's always required.  (I'm not sure if PropTypes has a way to require it depending on the value of the `type`.  The "correct" thing would be to require the `service` if and only if the `type` is "oauthError" but that might be getting too complicated.)